Library/Formula/beanstalk.rb

```diff
@@ -12,6 +10,7 @@ class Beanstalk < Formula
   end
 
   def install
+    mkdir_p var/"log"
```

Comment:
Try and see if they are included in bottles if that are in `install`. Otherwise move them all to `post_install`